libs/watt/src/lib/components/drawer/watt-drawer.component.ts

```diff
@@ -110,51 +130,40 @@ export class WattDrawerComponent implements OnDestroy {
     }
   }
 
-  /**
-   * Opens the drawer. Subsequent calls are ignored while the drawer is opened.
-   */
+  /** Opens the drawer. Subsequent calls are ignored while the drawer is opened. */
   open() {
-    // Trap focus whenever open is called. This doesn't work on the
-    // initial call (when first opening the drawer), but this is
-    // handled by the autoFocus property on mat-drawer.
-    this.cdkTrapFocus().focusTrap.focusInitialElementWhenReady();
-
-    // Disable click outside check until the current event loop is finished.
-    // This might seem hackish, but the order of execution is stable here.
-    this.bypassClickCheck = true;
-    setTimeout(() => {
-      this.bypassClickCheck = false;
-    }, 0);
-
-    if (!this.isOpen()) {
+    // Prevent tracking drawer signals when open is triggered in a reactive context
+    untracked(() => {
```

Comment:
suggestion: let's make the comment more explicit by adding some examples of "reactive context". I can only think of `effect` and `afterRenderEffect`. I hope noone calles `open` from a `computed`.

Reply:
Improved it a bit: c5b5b1002a4fcb18989946f65308e547fdb291a0